WooCommerce/Classes/ViewRelated/Themes/ProfilerThemesPickerView.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+import Kingfisher
+
+struct ProfilerThemesPickerView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+
+        ScrollView {
+            VStack(alignment: .leading) {
+                Spacer()
+                    .frame(height: Layout.contentVerticalSpacing)
+
+                Text(Localization.chooseThemeHeading)
+                    .bold()
+                    .largeTitleStyle()
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Spacer()
+                    .frame(height: Layout.contentPadding)
```

Comment:
Instead of adding spacer with fixed height, we can simply add vertical paddings to the `Text`:

```suggestion
                Text(Localization.chooseThemeHeading)
                    .bold()
                    .largeTitleStyle()
                    .padding(.horizontal, Layout.contentPadding)
                    .padding(.top, Layout.contentVerticalSpacing)
                    .padding(.bottom, height: Layout.contentPadding)
```

Reply:
Nice! Much more shorter for the same effect.